channels/rail/client/rail_orders.c

```diff
@@ -261,14 +261,14 @@ static UINT rail_read_server_get_appid_resp_order(wStream* s,
 	if (!s || !getAppidResp)
 		return ERROR_INVALID_PARAMETER;
 
-	if (Stream_GetRemainingLength(s) < 516)
+	if (Stream_GetRemainingLength(s) < 524)
```

Comment:
sure this will not break older servers?

Reply:
To my understanding this was a documentation error and the PDU actually contained 520 bytes of application id data from the beginning. Inf fact the current spec is still erroneous as it still shows the PDU with 512 bytes in the Protocol Examples chapter.

Reply:
Just checked legacy rail to a Win7 machine and it definitely sends a 520 bytes application id.

Reply:
ah, thanks for the clarification.